vulcan/lib/client/jsx/components/workflow/user_interactions/inputs/nested_select_autocomplete.tsx

```diff
@@ -53,7 +53,7 @@ function LeafOptions({
   handleSelect: (value: string | null, depth: number) => void;
   maxOptions?: number;
 }) {
-  if (!options_in) return null;
+  if ( (!options_in) || (value!=null && !options_in.includes(value)) ) return null;
```

Comment:
Checks if provided value exists in the given options to prevent MUI complaint bug